Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -73,6 +86,20 @@ - (NSString *)stringDescription
 	return self.variants.array ?: @[];
 }
 
+- (void)updateMinimumPrice
+{
+	NSDecimalNumber *newMinimumPrice = [self.variants valueForKeyPath:@"@min.price"];
+	if (![self.minimumPrice isEqual:newMinimumPrice]) {
+		self.minimumPrice = newMinimumPrice;
+	}
+}
+
+- (void)willSave
+{
+	[super willSave];
+	[self updateMinimumPrice];
```

Comment:
`updateMinimumPrice` is called from `setVariants:` as well. Curious to know if there's ever a case where the price won't be updated if we eliminate `willSave`.